addons/sale/views/sale_views.xml

```diff
@@ -391,6 +391,8 @@
                                 <group string="Reporting" name="technical" groups="base.group_no_one">
                                     <field groups="base.group_no_one" name="origin"/>
                                 </group>
+                                <group name="utm_link" groups="base.group_no_one">    
```

Comment:
Empty groups are not a good idea

Reply:
I need to add utm.mixin dependency in sale order to trace records of utm fields in my website_sale_order to generate utm chart in website dashboard. After adding dependency I need to update sale_order_form_view for utm fields so a user can see that utm fields in debug mode but module sale_crm also inherit utm.mixin and update sale_order_form_view for utm_fields so utm fields have a redundant view in sale order instead of this I add an empty group in sale_order_form_view in sale module and replace the view in website_link_tracker and sale_crm so there is no redundancy of utm_fields in sale_order view because whichever module is installed it replaces the whole group. Thanks